### PR TITLE
Merge sync-routerTemplates branch

### PR DESCRIPTION
**Pull request by vidurb**
Contains addition of templates for details, lister & teaser routers.

### CODE REVIEW

**Review by rungta — CHANGES_REQUESTED** [review]
We should probably add a check for `collection` being a non-empty list in the lister router.

**Comment by vidurb** [comment]
So far, that check has been in the listers, no?

**Comment by rungta** [comment]
Sure, but adding that check to the router directly makes it more robust and allows the caller to be that much more carefree.

**Comment by vidurb** [comment]
Added.

File: templates/_routers/lister.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% spaceless %}
+  {#
+    Lister Router
+  #}
+  {% if collection|length %}
```
Comment (rungta) [inline templates/_routers/lister.twig:6]:
```suggestion
  
  {% if collection|length %}
```

**Review by rungta — APPROVED** [review]

**Pull request merged**